### ISSUE TITLE
assert torch.all(0 <= bbox) and torch.all(bbox <= 1023)

### ISSUE THREAD

**Issue by minhson-kaist** [microsoft/unilm#913]
I am using LayoutLMv3 model to fine-tune on a custom FUNSD-like dataset and I got the error:
`assert torch.all(0 <= bbox) and torch.all(bbox <= 1023)
AssertionError`

This probem happened when I used the script `unilm/layoutlmv3/examples/run_funsd_cord.py`.
As I understand from the code, there was at least 1 value in the tensor `bbox` that is > 1023.

But when I check the `normalize_bbox` function in `unilm/layoutlmv3/layoutlmft/data/image_utils.py`, this function already called, it means the `bbox` values are already normalized.

I wonder if there is anything else can cause this error.

**Expected behavior**
- Platform: Ubuntu
- Python version: 3.7
- PyTorch version (GPU?): 1.10.0+cu111

**Comment by HYPJUDY** [comment]
This situation can happen when the logic of `assert torch.all(0 <= bbox) and torch.all(bbox <= 1023)` and `normalize_bbox` is not identical. You may check your data with `assert torch.all(0 <= bbox) and torch.all(bbox <= 1023)` right after `normalize_bbox` to see whether the error will occur.
